h/static/extension/config.js

```diff
@@ -1,8 +0,0 @@
-function hypothesisConfig() {
-  // Pages on our site can include a meta tag to trigger specific behaviour
-  // when the extension loads.
-  var hypothesisIntent = document.querySelector('[name="hypothesis-intent"]');
```

Comment:
So there were two different mechanisms here that could trigger the sidebar to show the login dialog when it opened, one was to provide a `hypothesisConfig()` function with a 'firstRun' property, the other was to add a `hypothesis-intent` meta tag to the page.

The reason for this second option was presumably so that it would work if `annotator/main.js` was run in an isolated JS world (possibly in the old Firefox extension) which couldn't see the page's JS.

I think we may want to look at doing that again in future, but we're not currently using it so I've modified `help.html.jinja` to always use the `hypothesisConfig()` method for the moment.